src/url.rs

```diff
@@ -29,6 +36,12 @@ impl Scanner for UrlScanner {
 impl UrlScanner {
     // See "scheme" in RFC 3986
     fn find_start(&self, s: &str) -> Option<usize> {
+        // Match protocol relative URLs (`//example.org`)
+        // See https://stackoverflow.com/a/15146073/270334
+        if s.is_empty() && self.trigger == Trigger::Slash {
```

Comment:
Hmm does this mean in `abc //example.org` we wouldn't match `//example.org`? Can you add a test case for this?

Also needs some more negative test cases, e.g. something like `foo//bar` should *not* match, but e.g. `"//foo.bar` should.